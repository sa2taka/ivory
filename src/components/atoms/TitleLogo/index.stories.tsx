import React from 'react';
import { TitleLogo as Component } from './index';
export default {
  title: 'atom',
};
export const TitleLogo = () => <Component />;
