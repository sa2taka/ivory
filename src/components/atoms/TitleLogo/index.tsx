import React from 'react';
import 'public/ivory-logo.png';

interface Props {
  className?: string;
}

export const TitleLogo: React.FC<Props> = ({ className }) => {
  return (
    <div className={`flex items-center ${className}`}>
      <img src="ivory-logo.png" className="h-8 w-8"></img>
      <span className="ml-2 text-2xl font-semibold">Ivory</span>
    </div>
  );
};
