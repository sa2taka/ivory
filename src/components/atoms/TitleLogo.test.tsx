import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { TitleLogo } from './TitleLogo';

describe('TitleLogo', () => {
  test('is rendered', () => {
    const wrapper = shallow(<TitleLogo />);
    expect(wrapper).toBeTruthy();
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
