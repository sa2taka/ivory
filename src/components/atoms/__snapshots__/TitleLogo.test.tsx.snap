// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleLogo is rendered 1`] = `
<div
  className="flex ml-4 items-center"
>
  <img
    className="h-8 w-8"
    src="ivory-logo.png"
  />
  <span
    className="ml-2 text-2xl font-semibold"
  >
    Ivory
  </span>
</div>
`;
